### PR TITLE
Update FB and Flickr importers to be similar in names and descriptions

### PR DESCRIPTION
**Pull request by mikegray**

### CODE REVIEW

File: samples/js/fb/find-photos/src/main.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ import {
 const args = argv
   .usage(
     'Finds photos in slurped Facebook metadata\n\n' +
-    'Usage: fb/find-photos <in-object> <out-dataset>')
+    'Usage: noms-facebook-find-photos <in-dataset> <out-dataset>')
```
Comment (arv) [inline samples/js/fb/find-photos/src/main.js]: Is this correct? We do not have a bin section in package.json so there is no binary installed.l To run this we need to do:

```sh
node . <in-dataset> <out-dataset>
```

Comment (mikegray) [inline samples/js/fb/find-photos/src/main.js]: ok, fixed all 4 places that have this issue. PTAL

File: samples/js/fb/find-photos/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "noms-fb-find-photos",
-  "description": "Finds photos in slurped flickr data",
+  "name": "noms-facebook-find-photos",
```
Comment (aboodman) [inline samples/js/fb/find-photos/package.json]: I think this can stay as `fb`.

Comment (mikegray) [inline samples/js/fb/find-photos/package.json]: ok, changed it back, but then changed noms-facebook-slurp to be noms-fb-slurp to match.

**Comment by mikegray** [comment]
PTAL

**Pull request merged**